src/schema/v2/partner.ts

```diff
@@ -238,6 +239,16 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
           sort: {
             type: ShowSorts,
           },
+          status: {
+            type: EventStatus.type,
+            defaultValue: "CURRENT",
```

Comment:
I think the default here should be the enum value, so "current" ? Then you might not need the special case below.